csharp/ql/test/query-tests/Security Features/CWE-601/UrlRedirect/UrlRedirect.expected

```diff
@@ -4,15 +4,12 @@ edges
 | UrlRedirect.cs:37:44:37:66 | access to property QueryString : NameValueCollection | UrlRedirect.cs:37:44:37:74 | access to indexer |
 | UrlRedirect.cs:38:47:38:69 | access to property QueryString : NameValueCollection | UrlRedirect.cs:38:47:38:77 | access to indexer |
 | UrlRedirectCore.cs:13:44:13:48 | value : String | UrlRedirectCore.cs:16:22:16:26 | access to parameter value |
-| UrlRedirectCore.cs:13:44:13:48 | value : String | UrlRedirectCore.cs:19:44:19:48 | access to parameter value : String |
-| UrlRedirectCore.cs:13:44:13:48 | value : String | UrlRedirectCore.cs:25:46:25:50 | access to parameter value : String |
+| UrlRedirectCore.cs:13:44:13:48 | value : String | UrlRedirectCore.cs:19:44:19:48 | call to operator implicit conversion |
```

Comment:
@hvitved : This looks like some kind of edge contraction (where the access and the implicit operator call now have been turned into a single edge), but that the equivalent flows have been identified. Do you know more about this?

Reply:
I think this is because the conversion operator is now in a stub, and therefore considered `not fromSource()`, and hence considered an [ordinary taint step](https://github.com/github/codeql/blob/main/csharp/ql/lib/semmle/code/csharp/dataflow/internal/TaintTrackingPrivate.qll#L76), which does indeed get contracted when possible.

Reply:
Great, thank you!